exercises/practice/pov/.docs/instructions.append.md

```diff
@@ -3,29 +3,42 @@
 ## Implementation Notes
 
 POV / reparent / change root of a tree
```

Comment:
I don't think so. I just kept it from the previous version. 
I have now also removed the next line "POV/...".
Please let me know if you would like to keep that.

Reply:
It looks like you also removed "## Implementation Notes". This line is needed because otherwise the append content is rendered directly below the instructions from problem spec. Depending on what the last sentence/section is there, that can lead to very confusing texts. (We had this issue with other exercises before.)

Reply:
Sorry, I did not mean to remove that line.